staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4298,6 +4298,15 @@ type PodStatus struct {
 	// +featureGate=DynamicResourceAllocation
 	// +optional
 	ResourceClaimStatuses []PodResourceClaimStatus `json:"resourceClaimStatuses,omitempty" patchStrategy:"merge,retainKeys" patchMergeKey:"name" protobuf:"bytes,15,rep,name=resourceClaimStatuses"`
+
+	// TotalRequestedResources is the effective resource requests for this pod taking into consideration init
+	// and regular containers, and the pod overhead. If a resize of pod resources has been requested for a running
+	// pod, this field reflects the total desired resources for that pod which differs from actual resources
```

Comment:
This sentence is a bit awkward, and reading it I'm not clear what the field does.

Does it:
1. Reflect the current allocated resource state on the kubelet?
2. Reflect the total desired resources for the pod?

Those are two different things, and I can't figure out which one it is.

Reply:
It's 2, I'll updatt the comment to clarify.